helm/ffc-pay-batch-verifier/values.yaml

```diff
@@ -18,13 +18,9 @@ containerSecret:
   type: Opaque
 
 deployment:
```

Comment:
So because removing `replicas` would result in `deployment` having no properties under it Helm doesn't like this.

Solution is to use an empty object, i.e. `deployment: {}`

Not sure if we could _remove_ the `deployment` key if there's nothing under it -- perhaps the Helm linter would look for this, perhaps the `deployment.yaml` wouldn't know where to bind to, perhaps it's best for readability and familiarity.